index.js

```diff
@@ -131,22 +136,24 @@ class ClientServiceBase {
       );
     }
 
-    const request = new FunctionRegistrationRequestBuilder(
+    const builder = new FunctionRegistrationRequestBuilder(
         new this.protobuf.FunctionRegistrationRequest(),
         this.signer).withFunctionId(id).
```

Comment:
I don't think signer is needed here, right ?

Reply:
Ah, Yes. It is redundant. My bad.